src/app/knob_page.component.ts

```diff
@@ -599,8 +632,10 @@ export class KnobPageComponent implements AfterViewInit, OnChanges {
     this.settingsButtonClicked.emit(openSettingsEvent);
   }
 
-  openLink(url: string) {
+  openLink(url: OutgoingLinkName) {
```

Comment:
Seems like we're adding typechecking here, but `openLink` is used with other URLs other than the 4 specified in the type, for example perspectiveapi.com, our Github org, the Google privacy policy, etc.:
https://github.com/conversationai/perspective-viewership-extension/search?q=openLink&unscoped_q=openLink

I would have expected the compiler to complain - do you know why it doesn't?

Do you know what happens if `openLink` is called with a string that's not included in the `OutgoingLinkName` type? It would be bad if there's a runtime error.

Reply:
Ohh, I see that there are actually multiple `openLink` functions. Maybe we can rename this one to be more specific, like `disabledPageOpenLink` or something?

Reply:
I updated the function name.